### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by YoshiS214**
Please change streamPage.dart
AppTabBar should be used for home of materialapp in main.dart

### CODE REVIEW

File: the_board/lib/main.dart [hunk 1]
```diff
@@ -50,6 +52,7 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
+  /*
```
Comment (zst-c) [inline the_board/lib/main.dart]: We can just delete this stuff

File: the_board/lib/pages/workroomPage.dart [hunk 2]
```diff
@@ -0,0 +1,378 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/cupertino.dart';
+
+class WorkroomPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      initialRoute: '/',
```
Comment (zst-c) [inline the_board/lib/pages/workroomPage.dart]: can we change the name of the route from / to something like /chatsList (or similar)?

File: the_board/lib/pages/workroomPage.dart [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+            TargetPlatform.iOS: CupertinoPageTransitionsBuilder(),
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class WorkRoomList extends StatelessWidget {
+  final List workrooms = [
```
Comment (zst-c) [inline the_board/lib/pages/workroomPage.dart:47]: Great - will get back to you on how we get that from the messaging group

File: the_board/lib/pages/workroomPage.dart [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+                          textAlign: TextAlign.left,
+                          maxLines: 1,
+                        ),
+                      ),
+                      Material(
+                        child: Text(
+                          room[3],
+                          maxLines: 2,
+                          style:
+                              TextStyle(color: Colors.grey[700]), // Last chat
```
Comment (zst-c) [inline the_board/lib/pages/workroomPage.dart:121]: Ideally at some point for all this theming, we'll write a themes.dart bit to keep consistency

**Review by zst-c — CHANGES_REQUESTED** [review]
Looks great! But please can you add comments to the code.

**Comment by YoshiS214** [comment]
I changed some parts and added some comments.
Is this a little bit better?

**Review by zst-c — APPROVED** [review]

**Pull request merged**